src/host_shaders/opengl_fragment_shader.frag

```diff
@@ -38,6 +38,21 @@ vec4 tevSources[16];
 vec4 tevNextPreviousBuffer;
 bool tevUnimplementedSourceFlag = false;
 
+// Holds the enabled state of the lighting samples for various PICA configurations
+// As explained in https://www.3dbrew.org/wiki/GPU/Internal_Registers#GPUREG_LIGHTING_CONFIG0
+const bool samplerEnabled[9 * 7] = bool[9 * 7](
```

Comment:
Might be better to pack this bools into bitfields

Reply:
Done